code/datums/components/suicide_count.dm

```diff
@@ -0,0 +1,67 @@
+/*
+
+	A component designed to log and count who has suicided with the attached item.
+	It can either be attached to an item like usual, or will be added automatically with NONE when someone suicides.
+	If view_mode is HOLY, then only those spiritual enough are able to detect the ghosts' spirits.
+	If it's ALL, any old folk can come and examine for the suicide information.
+	Forensics scanners can always detect an item's suicide counts. (not implemented yet todo)
+*/
```

Comment:
This isn't actually documenting the datum, I'm on mobile or id make a suggestion for you but just look at other code for an example